src/components/ShortGoalsComponent/ShortGoalsComponent.tsx

```diff
@@ -28,13 +28,16 @@ const ShortGoalsComponent = () => {
         </TouchableOpacity>
       </View>
       {show ? (
-        <FlatList
-          data={Data}
-          renderItem={({ item }) => <Card item={item} />}
-          keyExtractor={(item) => item.id}
-          nestedScrollEnabled={true}
-          testID="flatlist"
-        />
+        <View testID='flatlist'>
```

Comment:
why Flatlist is removed and map is used?

Reply:
Flatlist was not allowing me to put a scrollView. Then I tried to make the flatlist component scrollable, it was working when I only had one component but after I added this animated component it stopped working, it was working only with current solution